apps/04-golang-pgsql/Dockerfile

```diff
@@ -1,15 +1,23 @@
-FROM public.ecr.aws/p0k3r4s4/golang:1.19-alpine as build
+ARG GO_BASE_IMAGE=golang:1.19-alpine
```

Comment:
this is going to cause issues when this is run on CP app int as AWS will get rate limit issue from DH. This is why I changed it to point to our image on ECR.